src/shared/Internal.Shared.Tasks/Internal.Shared.Tasks.proj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
```

Comment:
Separately, it doesn't look like there's anything about this project that justifies having it in a "shared" folder.